spec/factories/students.rb

```diff
@@ -1,15 +1,11 @@
 FactoryBot.define do
-  factory :student do
-    title "Mr"
-    first_name "John"
-    middle_name "Adam"
-    last_name "Smith"
-    email "tester@example.com"
-    birth_date "1994-10-10"
-    gender "male"
-
-    trait :over_18 do
-      birth_date 18.years.ago
-    end
+  factory :student do |f|
+    f.title { Faker::Name.prefix}
+    f.first_name { Faker::Name.first_name }
+    f.middle_name { Faker::Name.first_name }
+    f.last_name { Faker::Name.last_name }
+    f.email { Faker::Internet.email }
+    f.birth_date { Faker::Date.birthday }
+    f.gender { Faker::Gender.type }
```

Comment:
I'm not sure you need the `f` here, you should be able to do something like 

```suggestion
  factory :student do
    title { Faker::Name.prefix}
    first_name { Faker::Name.first_name }
    middle_name { Faker::Name.first_name }
    last_name { Faker::Name.last_name }
    email { Faker::Internet.email }
    birth_date { Faker::Date.birthday }
    gender { Faker::Gender.type }
```

+1 for using Faker though 👍

Reply:
My mistake, I was following someone else's documentation without my thinking cap on. I've pushed some changes.